api/src/main/java/me/lokka30/treasury/api/economy/account/Account.java

```diff
@@ -38,14 +38,6 @@
  */
 public interface Account {
 
-    /**
-     * Gets the string-based unique identifier for this account.
-     *
-     * @return The String unique identifier for this account.
-     * @since v1.0.0
-     */
-    @NotNull String getIdentifier();
-
```

Comment:
Should we undo this change and instead make `getIdentifier()` use the `Object` type? Every account needs a way to identify itself so it would be nice if we could somehow force that in the interface. No worries if this is not feasible.

Reply:
the raw account type shouldn't be really used imo, any specific interactions should be done via the appropriate interface type.

Reply:
we should make sure to clarify that the account interface does not support custom implementations